cli/src/pcluster/api/util.py

```diff
@@ -148,3 +149,16 @@ def _deserialize_dict(data, boxed_type):
     :rtype: dict
     """
     return {k: _deserialize(v, boxed_type) for k, v in six.iteritems(data)}
+
+
+def assert_node_executable(logger):
+    node_exe = shutil.which("node")
+    logger.debug("Found nodejs executable in %s", node_exe)
+    if not node_exe:
+        message = (
+            "You must install Node.js in order to use ParallelCluster. Please install Node.js from https://nodejs.org"
```

Comment:
Adding the specific version of node in the message requires us to change the code every now and then to keep this help update so let's use a more generic message or better point to CDK documentation.

I'd say: 
```
Node.js is required to use ParallelCluster. 
It is a prerequisite of AWS CDK library used by ParallelCluster, 
see installation instructions here: https://docs.aws.amazon.com/cdk/latest/guide/work-with.html#work-with-prerequisites
```

Reply:
@nstornetta what do you suggest?

Reply:
Changed according latest discussion in the team